p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift

```diff
@@ -58,68 +54,90 @@ struct SellInputView: View {
     // MARK: - Subviews
 
     var baseAmountInputView: some View {
-        VStack(alignment: .leading, spacing: 4) {
-            sellAllButton
-                .padding(.leading, 24)
+        VStack(alignment: .leading, spacing: 18) {
+            HStack {
+                switchButton
+                Spacer()
+                sellAllButton
+            }
 
             HStack {
-                DecimalTextField(
-                     value: $viewModel.baseAmount,
-                     isFirstResponder: $viewModel.isEnteringBaseAmount
-                 ) { textField in
-                     textField.font = UIFont.font(of: .text3, weight: .regular)
-                     textField.keyboardType = .decimalPad
-                     textField.maximumFractionDigits = 2
-                     textField.decimalSeparator = "."
-                 }
-
-                 Text(viewModel.baseCurrencyCode)
-                    .apply(style: .text3)
-                    .foregroundColor(Color(Asset.Colors.night.color).opacity(0.3))
+                Text(viewModel.currentInputTypeCode)
+                   .apply(style: .largeTitle)
+                   .foregroundColor(Color(Asset.Colors.night.color))
+
+                Spacer()
+
+                if viewModel.showingBaseAmount {
+                    DecimalTextField(
+                        value: $viewModel.baseAmount,
+                        isFirstResponder: $viewModel.isEnteringBaseAmount
+                    ) { textField in
+                        textField.font = UIFont.font(of: .largeTitle, weight: .regular)
+                        textField.keyboardType = .decimalPad
+                        textField.maximumFractionDigits = 2
+                        textField.decimalSeparator = "."
+                        textField.textAlignment = .right
+                    }
+                } else {
+                    DecimalTextField(
+                         value: $viewModel.quoteAmount,
+                         isFirstResponder: $viewModel.isEnteringQuoteAmount
+                     ) { textField in
+                         textField.font = UIFont.font(of: .largeTitle, weight: .regular)
```

Comment:
`UIFont.font(of: .largeTitle, weight: .regular)` change to `.font(of: .largeTitle, weight: .regular)`